docs/architecture/psa-migration/strategy.md

```diff
@@ -256,6 +212,137 @@ Strategies currently (early 2022) used with each abstraction layer:
 - Cipher (G1): replace calls at each call site
 - MD (G1): replace calls at each call site
 
+
+Supporting builds with drivers without the software implementation
+==================================================================
+
+This section presents a plan towards G5: save code size by compiling out our
+software implementation when a driver is available.
+
+Additionally, we want to save code sive by compiling out the
+abstractions layers that we are not using when `MBEDTLS_USE_PSA_CRYPTO` is
+enabled (see previous section): MD and Cipher.
+
+Let's expand a bit on the definition of the goal: in such a configuration
+(driver used, software implementation and abstraction layer compiled out),
+we want:
+
+a. the library to build in a reasonably-complete configuration,
+b. with all tests passing,
+c. and no more tests skipped than the same configuration with software
```

Comment:
`No more` suggests that we're after numbers (so we can skip some, but add others), but I think that the idea here is to `have given features tested to the same extent using both configuration options`.

Reply:
It's clarified in the next paragraph, so I'm OK with that.